src/BenchmarkDotNet/Portability/RuntimeInformation.cs

```diff
@@ -200,7 +207,24 @@ internal static CpuInfo GetCpuInfo()
 
         internal static string GetRuntimeVersion()
         {
-            if (IsMono && !IsWasm)
+            if (IsWasm)
+            {
+                // code copied from https://github.com/dotnet/runtime/blob/2c573b59aaaf3fd17e2ecab95ad3769f195d2dbc/src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs#L20-L30
+                string versionString = typeof(object).Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion;
+
+                // Strip the git hash if there is one
+                if (versionString != null)
+                {
+                    int plusIndex = versionString.IndexOf('+');
+                    if (plusIndex != -1)
+                    {
+                        versionString = versionString.Substring(0, plusIndex);
+                    }
+                }
+
+                return $".NET Core (Mono) {versionString}";
```

Comment:
Is this really supposed to be mono in the string from IsWasm?

Reply:
Currently WASM is supported only by Mono VM.